src/Core/Migration/Migration1578590702AddedPropertyGroupPosition.php

```diff
@@ -0,0 +1,24 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Migration\MigrationStep;
+
+class Migration1578590702AddedPropertyGroupPosition extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1578590702;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $connection->executeUpdate('ALTER TABLE `property_group_translation` ADD `position` INT(11) NULL AFTER `description`;');
+    }
+
+    public function updateDestructive(Connection $connection): void
+    {
+        $connection->executeUpdate('ALTER TABLE `property_group_translation` DROP COLUMN `position`;');
```

Comment:
that's not meant for that

Reply:
Destructive updates are updates that execute irreversible schema changes. The idea is on a normal update you just do stuff that is compatible with older code changes. Any destructive change is about to break previous code. This is called blue-green deployment.

Reply:
... I'm so used to Symfony's up/down methods in the migrations that I didn't even read the method names. Thanks, will fix!